### DISCUSSION TITLE
Is there a way to set the background color or even make it alpha = 0?

### DISCUSSION THREAD

**Discussion by alanosman** [hvianna/audioMotion-analyzer#81] [Q&A] [answered]
I'm still learning about this great tool and got stuck.  I tried following the instructions but always got a black background.  I really want to make it alpha.  

Thanks

**Comment by tomsdob** [comment, marked as answer]
Was stuck with the same problem and found a fairly hard to find description of how to get the background transparent in the readme: https://github.com/hvianna/audioMotion-analyzer?tab=readme-ov-file#showbgcolor-boolean

Adding these made the background transparent:
```
showBgColor: false,
overlay: true,
```

**Comment by hvianna** [comment]
Check [`showBgColor`](https://audiomotion.dev/#/?id=showbgcolor-boolean) as it links to all other related properties.

I'm moving this to Discussions.
